printer_mods/DaGlo/Voron_V0_MW_PS_Bracket/README.md

```diff
@@ -0,0 +1,18 @@
+# Voron V0 Mean Well power supply bracket (Funssor Kit)
+
+This bracket works for the panel kit from Funssor from AliExpress, which has already some holes on the bottom bracket.
+
+"Funssor upgrade Voron 0 V 0,1 acryl pannel kit für Voron 0,1 3D drucker DIY kit"
+https://de.aliexpress.com/item/1005002596904062.html?spm=a2g0s.9042311.0.0.27424c4djYAPzu
```

Comment:
Please remove the session information from the link (everything after `.html`), it should be just `https://de.aliexpress.com/item/1005002596904062.html`